packages/integrations/image/components/Image.astro

```diff
@@ -0,0 +1,100 @@
+---
+// @ts-ignore
+import loader from 'virtual:image-loader';
+import { getImage } from '../src';
+import type { ImageAttributes, ImageMetadata, TransformOptions, OutputFormat } from '../src/types';
+
+export interface LocalImageProps extends Omit<TransformOptions, 'src'>, Omit<ImageAttributes, 'src'> {
+	src: ImageMetadata;	
```

Comment:
Is there already a way we could support this:

```astro
<Image src={import('../myImage.png')} />
```

in addition to the normal import?

```astro
---
import myImage from '../myImage.png';
---

<Image src={myImage} />
```

Understanding that we might add a utility directive to make this easier in the future.

Reply:
Actually, yes!  Just updated `Image.astro` to handle unwrapping the promise if the `import()` is inlined